plugins/org.jkiss.dbeaver.net.ssh.jsch/src/org/jkiss/dbeaver/model/net/ssh/SSHImplementationJsch.java

```diff
@@ -58,6 +58,13 @@ protected synchronized void setupTunnel(@NotNull DBRProgressMonitor monitor, @No
         if (jsch == null) {
             jsch = new JSch();
             JSch.setLogger(new JschLoggerProxy());
+            String serverHostKeys = JSch.getConfig("server_host_key");
+            String pubkeyAcceptedAlgorithms = JSch.getConfig("PubkeyAcceptedAlgorithms");
+            if (!CommonUtils.isEmpty(pubkeyAcceptedAlgorithms) && !CommonUtils.isEmpty(serverHostKeys) && !pubkeyAcceptedAlgorithms.contains(
+                "ssh-rsa")) {
+                JSch.setConfig("server_host_key", serverHostKeys + ",ssh-rsa");
+                JSch.setConfig("PubkeyAcceptedAlgorithms", pubkeyAcceptedAlgorithms + ",ssh-rsa");
```

Comment:
You use `"server_host_key"` and `"PubkeyAcceptedAlgorithms"` at least twice here. I think it's better to make a constant and use it instead.

Also, I'd make `ssh-rsa` to be a constant.